src/java/org/apache/cassandra/cql3/statements/SelectStatement.java

```diff
@@ -654,7 +657,8 @@ public ResultMessage.Rows executeInternal(QueryState state,
                                    userLimit,
                                    userPerPartitionLimit,
                                    pageSize,
-                                   aggregationSpec);
+                                   aggregationSpec,
+                                   true);
```

Comment:
walking this code its hard to understand stand what `local` means and I would leap to judgment about what it means... only once you fully walk to the thing that needs this boolean do you see

```
boolean allowsPotentialTxnConflicts,
```

leading this name and meaning to be confusing... 

https://cassandra.apache.org/_/development/code_style.html

```
Enums for Boolean Properties

Prefer an enum to boolean properties and parameters, unless clarity will be harmed (e.g. helper methods that accept a computed boolean predicate result, of the same name as used in the method they assist). Try to balance name clashes that would affect static imports, against clear and simple names that represent the behavioural switch.
```

enum could help in this case